backport/createStatusComment.ts

```diff
@@ -24,13 +24,17 @@ export const getCommentFromResponse = (pullNumber: number, backportResponse: Bac
   const helpParts = [];
 
   if (hasAnySuccessful) {
-    helpParts.push('Successful backport PRs will be merged automatically after passing CI.');
+    if (backportResponse.results.length === 1) {
+      helpParts.push('This backport PR will be merged automatically after passing CI.'); 
+    } else {
+      helpParts.push('Successful backport PRs will be merged automatically after passing CI.');
+    }
   }
 
   if (!backportResponse.success) {
     helpParts.push(
       [
-        'To backport manually, check out the target branch and run:',
+        'To backport manually run:',
```

Comment:
You can run the backport tool from any branch you like

Reply:
Yeah, I mostly added that message because we get a lot of questions that turn out to be someone trying to run backport from a branch with an old `.backportrc.json` file.

Reply:
Good point. This used to be a big problem but that should be fixed with https://github.com/sqren/backport/pull/245

Reply:
tldr: the .backportrc.json is retrieved from github.com instead of locally.

Reply:
Oh, I didn't realize! Cool, we don't need that messaging then.